src/main/java/mate/jdbc/dao/CarDaoImpl.java

```diff
@@ -0,0 +1,228 @@
+package mate.jdbc.dao;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.exception.DataProcessingException;
+import mate.jdbc.lib.Dao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.model.Car;
+import mate.jdbc.model.Driver;
+import mate.jdbc.model.Manufacturer;
+import mate.jdbc.util.ConnectionUtil;
+
+@Dao
+public class CarDaoImpl implements CarDao {
+    @Inject
+    private DriverDao driverDao;
+    @Inject
+    private ManufacturerDao manufacturerDao;
```

Comment:
you don't need to use these daos here, you should create queries that will be returning needed information about drivers and manufacturers here